CHANGELOG.md

```diff
@@ -1,5 +1,112 @@
 # Changelog
 
+### Breaking Change 
+
+In v0.2.0, `nonce` is required for `eea_sendTransaction`. Use [`priv_getTransactionCount`]( (https://docs.pantheon.pegasys.tech/en/latest/Reference/Pantheon-API-Methods/#priv_gettransactioncount))
+to calculate.
+
 ### Java 11
 
-EthSigner is supported on Java 11+. Java 8 support is deprecated and will be removed in a future release.
\ No newline at end of file
+EthSigner is supported on Java 11+. Java 8 support is deprecated and will be removed in a future release.
+
+## 0.2.0
+
+### Additions and Improvements 
+
+- [EthSigner documentation](https://docs.ethsigner.pegasys.tech/en/latest/)
+- Docker image [\#112](https://github.com/PegaSysEng/ethsigner/pull/112) 
+- Azure KeyVault \(cloud based software/HSM signing service\) signer support [\#109](https://github.com/PegaSysEng/ethsigner/pull/109) (thanks to [jimthematrix](https://github.com/jimthematrix))
+- Adding an UpCheck endpoint [\#96](https://github.com/PegaSysEng/ethsigner/pull/96) 
+- Hashi vault [\#87](https://github.com/PegaSysEng/ethsigner/pull/87) 
+- EEA sendTransaction [\#78](https://github.com/PegaSysEng/ethsigner/pull/78) 
+
+### Technical Improvements 
+
+- Release jar files to bintray [\#124](https://github.com/PegaSysEng/ethsigner/pull/124) 
+- Encode PrivateFrom/For into 32-byte strings [\#123](https://github.com/PegaSysEng/ethsigner/pull/123) 
```

Comment:
probably don't need this one - its invisible between releases, this only affects eeaPrivateTxn, which we didn't previously have (did we?) .. and if we did have it, then maybe this is simply (Correctly encoded eeaPrivateTransaction).
Is there a reason we link against the PR? _Most_ people won't care about that